### ISSUE TITLE
win-portable does not work on Win7 systems that presumably aren't more recently updated

### ISSUE THREAD

**Issue by NintendoManiac64** [SchildiChat/schildichat-desktop#99]
As of about a week ago, presumably due to matrix.org's use of 4096bit RSA keys, I've discovered that the vast majority of Matrix clients (including SchildiChat) no longer work on Windows 7 installations that presumably are not more recently updated (I've not yet been able to confirm its status on a more recently updated Win7 system), and even the app.element.io web client is impacted unless you use a Gecko-based browser (which presumably bundles its own copy of OpenSSL).  The attached screenshot is what appears when trying to launch the win-portable version of SchildiChat on such a not-recently-updated Win7 system.

In general I would not expect this to be something that would impact regular users on day-to-day systems and such, but the win-portable version of SchildiChat is likely the most heavily impacted Matrix client as one of the main benefits (if not entire point) of a portable application is to have it work regardless of the state of the OS and without the need to install something, especially when dealing with backup or troubleshooting situations.

Now I'm no software guru and for all I know this may be way out of the scope of this fork, but perhaps going the Gecko route of bundling your own copy of OpenSSL, if not at least for the win-portable version, could be a good idea.

I would imagine that the easiest way to replicate this is probably to just use a plain vanilla fresh, non-updated installation of Win7 SP1.

![screenshot](https://user-images.githubusercontent.com/11896003/135769193-f90c58a0-a741-4115-b334-110acb522faa.png)

**Comment by discapacidad5** [comment]
I found this information for Windows 7 update and the web version works perfectly for me I had the same problem with the web version before updating

Updated May 10, 2021 12:43

For devices running Windows 7, you must have the Microsoft™ 2019-09 Security Update installed. To learn more, see Microsoft's support article [ 2019 SHA-2 Code Signing Support requirement for Windows and WSUS.](https://support.microsoft.com/en-us/topic/2019-sha-2-code-signing-support-requirement-for-windows-and-wsus-64d1c82d-31ee-c273-3930-69a4cde8e64f)  

To apply the Microsoft 2019-09 Security Update, see the Windows 7 section of How to download a Windows update manually. If you have problems applying this security update, please contact [Microsoft Support.](https://support.microsoft.com/)

**Comment by su-ex** [comment]
Windows 7 is end of support, so probably not worth digging into any further.

**Comment by NintendoManiac64** [comment]
It is admittedly still getting paid support, but I do understand what you mean.

**Comment by su-ex** [comment]
I agree that the main benefit of a portable app is being portable and that there might be some Win 7 installations still around.
However, I have neither resources nor knowledge to do so: All I did was throwing together some scripting and what electron-builder spits out. I've only tested this on a VM. Honestly, I'm glad and surprised that the portable version still works.
If you or somebody else can give a hint on how to fix this, I'll be happy to incorporate it!

**Comment by discapacidad5** [comment]
Although they do not believe it, there is still windows xp one thing is
that microsoft does not support them something different that is not used.
most of the poor people have old pc so they can only use windows 7 some
windows xp, only in the first world everyone is up to date. Microsoft has
stopped supporting forcing people to buy new equipment, but most poor
people can't. free software should think more about people and not about
the capital that forces to update new equipment it is unfortunate that some
programs are no longer supporting windows 7 when there are a large number
of people who still use it
